FLEDGE_Key_Value_Server_API.md

```diff
@@ -129,6 +128,8 @@ In the request, one major difference from V1/BYOS is that the keys are now group
 
 ![request structure](assets/fledge_kv_server_v2_req_structure.jpeg)
 
+In addition, Chrome may only pass [contextual signals](https://github.com/WICG/turtledove/blob/main/PA_Key_Value_Server_Contextual_Signals.md) (`sellerTKVSignals` and `buyerTKVSignals`) to a trusted key/value service.
```

Comment:
Shouldn't mention Chrome - this a cross-browser proposal, not a Chrome-only feature.